alchemy/tsconfig.json

```diff
@@ -1,16 +1,17 @@
 {
   "extends": "../tsconfig.base.json",
-  "include": ["src/**/*"],
+  "include": ["src/**/*.ts"],
   "compilerOptions": {
     "composite": true,
     "noEmit": false,
     "outDir": "./lib",
     "rootDir": "./src",
     "sourceMap": true,
-    "declarationMap": true,
```

Comment:
Why not declaration map?

Reply:
We might not be making use of it properly right now. I don't know if we need it necessarily, but the outcome we want is:
1. users of `alchemy` get stack traces that point to the original `.ts`
2. jump to definition should bring the user to the `.ts` file (this may require packaging the `.ts` source in NPM too)